mhctools/cli/script.py

```diff
@@ -74,7 +83,10 @@ def main(args_list=None):
         input_dictionary = parse_fasta_dictionary(args.input_fasta_file)
         binding_predictions = predictor.predict_subsequences(input_dictionary)
     elif args.sequence:
-        binding_predictions = predictor.predict_peptides(args.sequence)
+        if args.extract_subsequences:
+            binding_predictions = predictor.predict_subsequences(args.sequence)
+        else:
+            binding_predictions = predictor.predict_peptides(args.sequence)
     elif args.input_peptides_file:
```

Comment:
Would you want to just combine peptides from input files and the commandline (e.g. allow `--sequence XYZ --input-peptides-file morepeps.txt`)?